presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,44 +28,60 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
-Requirements
-------------
+You should be using the same version of the JDBC driver as your Presto
+installation. If you are not on the latest release shown above, a list of all
+currently available versions can be found `here <https://search.maven.org/artifact/io.prestosql/presto-jdbc>`__.
```

Comment:
Dont use "here". Instead use "the Maven Central repository" 

And use this link in fact - https://repo1.maven.org/maven2/io/prestosql/presto-jdbc/ .. it allows browsing

Also mention that the folder contains all the different version and in each folder the driver file is named presto-jdbc-xxx.jar  .. with xxx the version number .. that is the file that needs to be downloaded .. not any of the others.